data/transactions/asset.go

```diff
@@ -75,3 +78,38 @@ type AssetFreezeTxnFields struct {
 	// AssetFrozen is the new frozen value.
 	AssetFrozen bool `codec:"afrz"`
 }
+
+var errAssetIDCannotBeZero = errors.New("asset ID cannot be zero")
+var errCannotCloseAssetByClawback = errors.New("cannot close asset by clawback")
+var errFreezeAccountCannotBeEmpty = errors.New("freeze account cannot be empty")
+
+// wellFormed performs some stateless checks on the AssetConfig transaction
+func (ac AssetConfigTxnFields) wellFormed() error {
+	return nil
+}
```

Comment:
I didn't see anything in the existing code to move here, since you can create an ASA with absolutely nothing defined and it works.